spec/apiForSheetsSpec.js

```diff
@@ -402,6 +404,7 @@ describe('Api For Sheets Module', () => {
       expect(checkWipModule.checkWIP).not.toHaveBeenCalled();
       expect(apiForSheetsModule.checkClaStatus).not.toHaveBeenCalled();
       expect(checkPullRequestJobModule.checkForNewJob).not.toHaveBeenCalled();
+      expect(checkPullRequestJobModule.checkForNewCronJob).toHaveBeenCalled();
```

Comment:
Why do we expect the `cronjob` check to have been called here?

Reply:
I was not sure as to where I have to add these tests, so I added it everywhere where `checkForNewJob` was called.